README.md

```diff
@@ -31,6 +31,7 @@ There are two main categories of samples: **Camera** and **Applications**. The s
     - [**CaptureHDRCompleteSettings**][CaptureHDRCompleteSettings-url] - Capture point clouds, with color, from the Zivid camera with fully configured settings.
   - **Advanced**
     - [**CaptureHDRLoop**][CaptureHDRLoop-url] - Cover the same dynamic range in a scene with different acquisition settings to optimize for quality, speed, or to find a compromise.
+    - [**CaptureHDRPrintNormals**][CaptureHDRPrintNormals-url] - Capture Zivid point clouds, calculate normals and print a subset.
```

Comment:
Can you use "compute"

"We use a calculator to perform simple arithmetic operations, whereas a computer is typically used to perform complicated tasks" from https://jakubmarian.com/difference-between-compute-and-calculate/#:~:text=We%20use%20a%20calculator%20to,%E2%80%9Ccomputation%E2%80%9D%20to%20indicate%20complexity.